examples/src/main/scala/libretto/examples/diningPhilosophers/Forks.scala

```diff
@@ -33,5 +33,5 @@ trait Forks {
     *  - signal readiness (when picked up or done being used).
     *  - defer readiness by awaiting a [[Done]] signal.
     */
-  implicit def heldForkReadiness: SignalingJunction.Positive[HeldFork]
+  given heldForkReadiness: SignalingJunction.Positive[HeldFork]
```

Comment:
I was very unsure to remove the (technically unneeded) name. But the comment could be better. The name is better than the description, imho. Could be slightly reworked, I guess

Reply:
Making the examples easier to understand _is_ important, but a task of its own.